packages/analytics-js-integrations/src/integrations/PinterestTag/browser.js

```diff
@@ -230,8 +230,9 @@ export default class PinterestTag {
     window.pintrk('track', event, pageObject);
   }
 
-  identify() {
-    const userTraits = this.analytics.getUserTraits();
+  identify(rudderElement) {
+    const { context } = rudderElement.message;
+    const userTraits = context?.traits || {};
```

Comment:
Don't we need to consider traits.email?

Reply:
no in js-sdk user traits or identify call should be referred from context.traits only and for group call we should refer from root level traits. no need to take it from traits.email